fvtest/tril/tril/CMakeLists.txt

```diff
@@ -23,44 +23,88 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../cmake/modules"
 	${CMAKE_MODULE_PATH})
 
 include(OmrUtility)
-include(FindYACC)
-include(FindLEX)
-
-find_package(YACC REQUIRED)
-find_package(LEX REQUIRED)
```

Comment:
I just want to point out that the REQUIRED option is actually not handled by these two find modules. I'll open an issue for this.